BraveWallet/Crypto/Stores/CryptoStore.swift

```diff
@@ -195,6 +196,13 @@ public class CryptoStore: ObservableObject {
     }
   }
   
+  func closeBSSStores() {
+    guard buySendSwapDestination == nil else { return }
```

Comment:
I'm not sure this check makes sense, since during the wallet panel buy/send/swap flow we never assign this `buySendSwapDestination` value in the `presentBuySendSwap` closure so it will always be nil. On the other hand if we do assign it, we'd have to clear it here rather than check against it. 🤔

I guess we should update the `didSet` for `buySendSwapDestination` to call this function though.